week10/MovieViewer/app/src/main/java/com/wmw/movieviewer/di/ViewModelModule.kt

```diff
@@ -0,0 +1,15 @@
+package com.wmw.movieviewer.di
+
+import com.wmw.movieviewer.ui.MovieDetailViewModel
+import com.wmw.movieviewer.ui.MovieViewModel
+import org.koin.androidx.viewmodel.dsl.viewModel
+import org.koin.dsl.module
+
+val viewModelModule = module {
+    viewModel {
+        MovieViewModel()
```

Comment:
You should pass your dependencies through the constructor instead of injecting or setting them in the ViewModels.